Sources/Fog/View/FogRngSet.swift

```diff
@@ -194,6 +211,7 @@ private final class RngTracker {
     var knownBlockCount: UInt64
 
     init(rng: FogRng, startBlockIndex: UInt64) {
+        logger.info("rng: \(rng), startBlockIndex: \(redacting: startBlockIndex)")
```

Comment:
`FogRng` needs to be redacted. Also not sure it prints anything useful at the moment since it's a wrapper around a rust class.